devito/dle/parallelizer.py

```diff
@@ -22,15 +22,49 @@ def nhyperthreads():
     return configuration['platform'].threads_per_core
 
 
-class NThreads(Constant):
+class NThreadsMixin(object):
+    pass
+
+
+class NThreads(Constant, NThreadsMixin):
+
+    name = 'nthreads0'
+    aliases = (name, 'nthreads')
 
     @classmethod
     def default_value(cls):
         return int(os.environ.get('OMP_NUM_THREADS', ncores()))
 
     def __new__(cls, **kwargs):
-        return super(NThreads, cls).__new__(cls, name=kwargs['name'], dtype=np.int32,
-                                            value=NThreads.default_value())
+        name = kwargs.get('name', NThreads.name)
+        value = NThreads.default_value()
```

Comment:
can't pass it in as a kwarg?

Reply:
no I'm using default_value() for testing too